Automattic/vip-go-mu-plugins
# Bump vip-support to 4aacd9f

Adds meta to support users so it's easier to identify them even if the role changes. Will also correctly remove these users.

## Checklist


## Steps to Test

Example:

1. Check out PR.
1. Add some support users, remove the meta for some of them, set the role for one to inactive.
1. Adjust the registered date to older than 8 hours.
1. Verify that running `Automattic\VIP\Support_User\User::init()->remove_stale_support_users()` removes them.
